pyiron/vasp/volumetric_data.py

```diff
@@ -150,6 +151,8 @@ def _read_vol_data(self, filename, normalize=True):
             list: A list of the volumetric data (length >1 for CHGCAR files with spin)
 
         """
+        if os.stat(filename).st_size == 0:
```

Comment:
To have the same behaviour when parsing empty files we should also raise a warning here: 
`s.logger.warning("File:" + filename + "seems to be corrupted/empty")`

Reply:
Fixed [here](https://github.com/pyiron/pyiron/pull/185/commits/8698b6641ca5b311f288b3a886e691eb69b0225f)